### ISSUE TITLE
Vertical zoom, Horizontal zoom

### ISSUE THREAD

**Issue by olablt** [liihuu/KLineChart#250]
Hello,

only Horizontal Zoom is working for me. Vertical is automatically adjusted.
How to Scroll / Zoom vertically? For example to see the nearest key support/ resistance lines?

![image](https://user-images.githubusercontent.com/674574/171670826-35eebf83-0e76-45b2-b8e7-3859432e3b6e.png)

![image](https://user-images.githubusercontent.com/674574/171671058-89d12653-5f15-44d5-ab9f-0a0e2cb6fce4.png)

**Comment by liihuu** [comment]
Y-axis scaling will be added in a later version.

**Comment by olablt** [comment]
hey, @liihuu , how are you? hope everything is good :) !

just wanted to let you know that this library is best of the best ;) !

maybe you have a debug version of manual Y-axis scaling? or maybe it could be possible to set the scaling in the code and move on the chart in any direction without auto scaling?

**Comment by liihuu** [comment]
I'm trying to build the next version, of course this feature will be in the next version.

**Comment by guelweles** [comment]
Was Y-axis scaling/scrolling already implemented? If not, any ideas when this would be available?

**Comment by liihuu** [comment]
It has come. The refactoring is almost complete. This feature has been added and will be released soon.

**Comment by liihuu** [comment]
V9 alpha has been released and supports this feature. You can try it.

**Comment by guelweles** [comment]
does it mean that we can manually scroll the Y axis? I did upgrade to v9 but I'm getting a lot of errors, I was able to sort some of them and finally could load the chat, however I saw the Y axis keeps being adjusted automatically as v8, not sure if I got it correctly, for example, in tradingview, one can hover the mouse on the Y axis and scroll it up and down.

For now I'll keep using v8 as my code broke in many parts, I'll get some time to carefully read the v9 documentation and try to upgrade it again, anyway I appreciate your amazing lib as always, keep it up.
